JSDemos/Demos/DataGrid/CustomDataSource/Vue/App.vue

```diff
@@ -60,8 +62,10 @@ const store = new CustomStore({
       'group',
       'groupSummary',
     ].forEach((i) => {
-      if (i in loadOptions && isNotEmpty(loadOptions[i])) {
-        params += `${i}=${JSON.stringify(loadOptions[i])}&`;
+      const property = i as LoadOptionsKey;
```

Comment:
better to remove `typeof` above and `as` here and specify that array is immutable. Then it will have proper type instead of `string[]`

```ts
([
  'skip',
  'take',
  'requireTotalCount',
  'requireGroupCount',
  'sort',
  'filter',
  'totalSummary',
  'group',
  'groupSummary',
] as const).forEach((i) => {...})
```